HeinOnline.js

```diff
@@ -0,0 +1,240 @@
+{
+	"translatorID": "3dcbb947-f7e3-4bbd-a4e5-717f3701d624",
+	"label": "HeinOnline",
+	"creator": "Frank Bennett",
+	"target": "https?://heinonline.org/HOL/(?:LuceneSearch|Page|IFLPMetaData)\\?",
+	"minVersion": "1.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsbv",
+	"lastUpdated": "2016-02-22 23:08:21"
+}
+
+/*
+    ***** BEGIN LICENSE BLOCK *****
+
+    Copyright © 2015-2016 Frank Bennett
+
+    This file is part of Zotero.
+
+    Zotero is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Zotero is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+    ***** END LICENSE BLOCK *****
+*/
+
+
+/*
+    ***************
+    ** Utilities **
+    ***************
+*/
+
+// Get any search results from current page
+// Used in detectWeb() and doWeb()
+function getSearchResults(doc, url) {
+	var results = doc.getElementsByClassName("lucene_search_result_b"),
+		items = {},
+		found = false
+	for (var i=0, ilen=results.length; i<ilen; i++) {
+		var url = getXPathStr("href", results[i], './/a[1]');
```

Comment:
And it's generally best to get the value of `.href` from the anchor element than the value of the href html attribute via xpath.  Usually it works out the same, but sometimes it may differ.